theano/gpuarray/extra_ops.py

```diff
@@ -430,8 +430,8 @@ def c_support_code_struct(self, node, nodename):
                         size_t dimBlock[3] = {1, 1, 1};
                         size_t tmp0 = shape[axis]-2;
                         size_t tmp1 = shape[axis]-1;
-                        void* kernel_params[] = {(void*) input->ga.data,
-                                                 (void*) output->ga.data,
+                        void* kernel_params[] = {(void*) ((char *)(input->ga.data) + input->ga.offset),
```

Comment:
Please do not use GA_POINTER.  It isn't functional and I'll get rid of it soon.  But otherwise, yes, you're right.

I must have been too addled when I merged this to miss that problem.